packages/vue/src/components/atoms/SfImage/SfImage.spec.js

```diff
@@ -1,13 +1,99 @@
 import { shallowMount } from "@vue/test-utils";
 import SfImage from "./SfImage.vue";
 describe("SfImage.vue", () => {
-  it("renders a component", () => {
-    const component = shallowMount(SfImage, {
+  it("renders component with defaults", () => {
+    const wrapper = shallowMount(SfImage);
+    expect(wrapper.classes("sf-image--wrapper")).toBe(true);
+  });
+  it("renders component with props", () => {
+    const wrapper = shallowMount(SfImage, {
       propsData: {
         alt: "test",
         src: "test",
       },
     });
-    expect(component.classes("sf-image--wrapper")).toBe(true);
+    expect(wrapper.classes("sf-image--wrapper")).toBe(true);
+  });
+  it("renders component with srcset", () => {
+    const wrapper = shallowMount(SfImage, {
+      propsData: {
+        srcsets: [
```

Comment:
same here (and below too) - even if we check only srcset we still need src and alt props :)